adi-colorimeter.desktop.in

```diff
@@ -1,5 +1,5 @@
 [Desktop Entry]
-Exec=gksudo -u root @PREFIX@/bin/adi_colorimeter
+Exec=pkexec -u root @PREFIX@/bin/adi_colorimeter
```

Comment:
Be aware that some lauchers might not work this IIRC. In osc we needed [this](https://github.com/analogdevicesinc/iio-oscilloscope/blob/master/osc-wrapper.cmakein). IIRC, the albert launcher was having this issue...

Reply:
Okay was not aware of that. Then the best thing would be to do the same here with a wrapper right ?

Reply:
If we still need it. If you have the time, maybe try with this https://github.com/albertlauncher/albert one... I think this was the one giving me issues...

Reply:
i already added the wrapper
if it is the safest path we can go with it.

Reply:
seems to work without the wrapper 
![image](https://user-images.githubusercontent.com/8693228/108968049-f3a15b00-7688-11eb-8729-5fbf7fff685c.png)

Reply:
![image](https://user-images.githubusercontent.com/8693228/108968122-0b78df00-7689-11eb-8d69-d69df60b9b75.png)